Prace_domowe/Praca_domowa2/Grupa2/Komorowski_Michal/notebook.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>Nie do końca o to chodziło -- celem było porównanie wartości imputowane z oryginalnymi, a Ty porównałeś wszystkie wartości (również te oryginalne), co zaniża RMSE. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>